src/input/password-input.less

```diff
@@ -0,0 +1,28 @@
+.password-input-component {
```

Comment:
Can't it simply take everything from the default input?

Reply:
True. Will take care of that.

Reply:
`fixed`